test.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+use Http\Discovery\Psr17FactoryDiscovery;
+use Symfony\Component\Serializer\Normalizer\UidNormalizer;
+
+require 'vendor/autoload.php';
```

Comment:
I would prefer this kind of file to be in a separated folder to avoid having too much in root folder :)

Reply:
I accidentally commit it with 2 others :laughing:

Reply:
Sorry for the mess; it's clean now